src/testUtils/logging.ts

```diff
@@ -1,33 +1,38 @@
-import pino, { symbols as PinoSymbols } from 'pino';
+import pino, { type LogDescriptor, symbols as PinoSymbols } from 'pino';
 import split2 from 'split2';
 
-export type MockLogSet = object[];
+type MockLogSet = object[];
 
 export interface MockLogging {
   readonly logger: pino.Logger;
   readonly logs: MockLogSet;
 }
 
 export function makeMockLogging(): MockLogging {
-  const logs: object[] = [];
+  const logs: any[] = [];
   const stream = split2((data) => {
     logs.push(JSON.parse(data));
   });
   const logger = pino({ level: 'debug' }, stream);
   return { logger, logs };
 }
 
-export function partialPinoLogger(bindings: { readonly [key: string]: any }): object {
+// eslint-disable-next-line import/no-unused-modules
+export function partialPinoLogger(bindings: { readonly [key: string]: any }): any {
```

Comment:
We'll need this utility later for sure, so I'm suppressing the error that it's unused.